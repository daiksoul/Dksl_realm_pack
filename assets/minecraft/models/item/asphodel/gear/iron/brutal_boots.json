{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/iron_boots",
		"layer1": "misc/brutal_symbol"
	}
}
