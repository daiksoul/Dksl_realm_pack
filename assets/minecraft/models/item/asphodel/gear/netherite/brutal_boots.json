{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/netherite_boots",
		"layer1": "misc/brutal_symbol"
	}
}
