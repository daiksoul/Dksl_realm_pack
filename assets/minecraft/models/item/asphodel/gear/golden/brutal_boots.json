{
	"parent": "item/handheld",
	"textures": {
		"layer0": "item/golden_boots",
		"layer1": "misc/brutal_symbol"
	}
}
